test/hotspot/jtreg/compiler/c2/TestFloat16Reduction.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package compiler.c2;
+
+/*
+ * @test
+ * @bug 8370409
+ * @summary Incorrect computation in Float16 reduction loop
+ *
+ * @library /test/lib /
+ * @run main/othervm -XX:-TieredCompilation
+ *                   compiler.c2.TestFloat16Reduction
```

Comment:
Was the flag required for reproducing the issue?
If it was not required: just remove it
If it was required: add a run without the flag, in addition to a run with the flag.

Reply:
My bad, we don't need these, looks like a copy-paste error, IR framework uses white box APIs